plugins/module_utils/network/os10.py

```diff
@@ -113,13 +113,18 @@ def load_config(module, commands):
     rc, out, err = exec_command(module, 'configure terminal')
     if rc != 0:
         module.fail_json(msg='unable to enter configuration mode', err=to_text(err, errors='surrogate_or_strict'))
-
+    if module.params['use_transaction']:
+        module.warn("Starting transaction")
+        commands.insert(0, 'start transaction')
     commands.append('commit')
```

Comment:
I'm wondering if I can move this line under the check for `use_transaction` and resolve #66  ?

Reply:
@gunasekaran-t what do you think?